ansible-deployment/ironic_config/tasks/main.yaml

```diff
@@ -15,6 +15,16 @@
          path: /var/lib/config-data/puppet-generated/ironic/etc/ironic/ironic.conf
          insertafter: ^# *rpc_response_timeout *=.*
          line: rpc_response_timeout = 600
+     - name: Set rescue kernel image
+       lineinfile:
+         path: /var/lib/config-data/puppet-generated/ironic/etc/ironic/ironic.conf
+         insertafter: ^# *rescue_kernel *=.*
+         line: rescue_kernel = {{ rescue_kernel }}
+     - name: Set rescue kernel ramdisk
+       lineinfile:
+         path: /var/lib/config-data/puppet-generated/ironic/etc/ironic/ironic.conf
+         insertafter: ^# *rescue_ramdisk *=.*
+         line: rescue_kernel = {{ rescue_ramdisk }}
```

Comment:
Should it be `line: rescue_ramdisk = {{ rescue_ramdisk }}`?

Reply:
100% - thanks for catching that. Fixed!